modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/renderer/v2_0/DataLayoutRendererImpl.java

```diff
@@ -153,7 +148,4 @@ private DDMFormRenderingContext _toDDMFormRenderingContext(
 	@Reference
 	private DDMStructureVersionLocalService _ddmStructureVersionLocalService;
 
-	@Reference
-	private Portal _portal;
```

Comment:
@JanisZhang `LocaleUtil.fromLanguageId` has logic calling `LanguageUtil` and possibly `PortalUtil`. Please exclude this field, record it, and resend, thanks!